cluster-install/install.sh

```diff
@@ -1637,13 +1642,30 @@ step_check_all_images_can_be_pulled() {
         if config_connector_operator_image_is_custom; then images_to_check+=("$custom")
         else images_to_check+=("$default"); fi
 
-        images_to_check+=("$CONNECTOR_IMAGE_REGISTRY" "$CONNECTOR_IMAGE_REPO" "$CONNECTOR_IMAGE_TAG")
-        if config_connector_image_is_custom; then images_to_check+=("$custom")
-        else images_to_check+=("$default"); fi
 
-        images_to_check+=("$NEPTUNE_IMAGE_REGISTRY" "$NEPTUNE_IMAGE_REPO" "$NEPTUNE_IMAGE_TAG")
-        if config_neptune_image_is_custom; then images_to_check+=("$custom")
-        else images_to_check+=("$default"); fi
+        if config_connector_image_is_custom; then
+          # Only check connector image if user has overridden the connector-operator's default.
+          # We do not know the default version and cannot check it due to it being hard coded within the connector-operator image.
+          images_to_check+=("$CONNECTOR_IMAGE_REGISTRY" "$CONNECTOR_IMAGE_REPO" "$CONNECTOR_IMAGE_TAG" "$custom")
+        else
+          # Get the default connector tag
+          file_content=$(curl -sS "https://raw.githubusercontent.com/NetApp/astra-connector-operator/$CONNECTOR_OPERATOR_IMAGE_TAG/common/connector_version.txt")
+          # Trim new lines and white space
+          tag="${file_content//[[:space:]]/}"
```

Comment:
Also just realized -- should probably make these two variables (file_content, tag) local to the function (unless I missed their declaration somewhere), as they're gonna end up being global for the whole script. 

Important notes about local bash variables though is that they're scoped to the entire function, even if declared within an `if` block; so if you declare it readonly `local -r my_readonly_var` in multiple places in the same function, you'll probably get an error.